.gitignore

```diff
@@ -39,6 +39,8 @@ benchmark/mysql-connector-cpp/
 benchmark/mariadb.json
 benchmark/mysql.json
 benchmark/main-benchmark
+cmake-build-*/
```

Comment:
I gave some comments in the PR you referred. I'm gonna repeat them here.
These are too project-specific. I mean, I don't think they are default names of directories created by cmake/build by cmake generated projects.  I am not sure I want these to be added to .gitignore for everybody. On other hand it's pretty common naming pattern. Once again - I am not sure about this